tests/functional/c/consider/consider_iterating_dictionary.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-docstring, expression-not-assigned, too-few-public-methods, no-member, import-error, no-self-use, line-too-long, useless-object-inheritance, unnecessary-comprehension
+# pylint: disable=missing-docstring, expression-not-assigned, too-few-public-methods, no-member, import-error, no-self-use, line-too-long, useless-object-inheritance, unnecessary-comprehension, use-dict-literal
 
```

Comment:
```suggestion
# pylint: disable=missing-docstring, expression-not-assigned, too-few-public-methods
# pylint: disable=no-member, import-error, no-self-use, line-too-long, useless-object-inheritance
# pylint: disable= unnecessary-comprehension, use-dict-literal

```
Nitpick: As we're changing the whole txt result already we might as well reduce the lenght of the disable line too :)

Reply:
See latest commit :)